src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -260,6 +266,11 @@ private static function normalizeQueueArguments(array $arguments): array
         return $arguments;
     }
 
+    private static function hasCaCertConfigured(array $amqpOptions): bool
+    {
+        return (isset($amqpOptions['cacert']) && '' !== $amqpOptions['cacert']) || '' !== ini_get('amqp.cacert');
```

Comment:
I don't understand where the value of amqp.cacert is used.

Reply:
The value of amqp.cacert is used by php amqp library, in fact it does load parameter from env, conf or ini. One of least should be specificed and they are loaded in $amqpOptions['cacert']